### PR TITLE
upgrade tidb to v5.2.0 in e2e

### PR DESCRIPTION
**Pull request by KanShiori**
### What problem does this PR solve?

upgrade tidb to v5.2.0 in e2e
### Code changes

- [ ] Has Go code change
- [x] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [ ] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: tests/e2e/util/image/image.go [hunk 1]
```diff
@@ -33,14 +33,13 @@ var (
 const (
 	// TiDB Version
 	TiDBV3             = "v3.0.20"
-	TiDBLatestPrev     = "v5.0.3"
-	TiDBLatest         = "v5.1.0"
+	TiDBLatestPrev     = "v5.1.1"
```
Comment (DanielZhangQD) [inline tests/e2e/util/image/image.go:36]: 5.1.1 needs to be covered in L30.

Comment (KanShiori) [reply tests/e2e/util/image/image.go:36]: 1063aa8067e6613585d68f96fdd73c90a9df15c2

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**